src/components/RegistrationForm/RegistrationForm.tsx

```diff
@@ -0,0 +1,247 @@
+          />
+          <FormSelect
+            label={languageMode == "english" ? "Degree" : "Stopień studiów"}
+            registerName="degree"
+            isRequired={true}
+            options={
+              languageMode == "english" ? degreeOptions.EN : degreeOptions.PL
+            }
+          />
+          <FormSelect
+            label={languageMode == "english" ? "Session" : "Semestr"}
+            registerName="session"
+            isRequired={true}
+            options={sessionOptions.PL}
+          />
+        </div>
+        <Typography variant="h6" className="text-primary-color m-4">
+          {languageMode == "english"
+            ? "Additional questions"
+            : "Dodatkowe informacje"}
+        </Typography>
+        <div className="flex flex-col md:grid md:grid-cols-3">
+          <FormSelect
+            label={languageMode == "english" ? "Diet" : "Dieta"}
+            registerName="diet"
+            isRequired={true}
+            options={
+              languageMode == "english" ? dietOptions.EN : dietOptions.PL
+            }
+          />
+          <FormSelect
+            registerName="shirtSize"
+            label={
+              languageMode == "english" ? "T-shirt size" : "Rozmiar koszulki"
+            }
+            isRequired={true}
+            options={shirtSizeOptions.PL}
+          />
+          <FormField
+            label={languageMode == "english" ? "Foot size" : "Rozmiar stopy"}
+            registerName="footSize"
+            isRequired={true}
+            minLength={2}
+            maxLength={20}
+            fieldType="numeric"
+          />
+          <FormSelect
+            label={
+              languageMode == "english"
+                ? "How do you know about Wtyczka"
+                : "Skąd wiesz o Wtyczce"
+            }
+            registerName="source"
+            isRequired={true}
+            options={
+              languageMode == "english" ? sourceOptions.EN : sourceOptions.PL
+            }
+          />
+          <FormSelect
+            label={languageMode == "english" ? "Invoice" : "Faktura"}
+            registerName="invoice"
+            isRequired={true}
+            options={
+              languageMode == "english" ? invoiceOptions.EN : invoiceOptions.PL
+            }
+          />
+        </div>
+        <div className="flex flex-col m-4">
+          <FormCheckbox label={statuteConsentLabel} name="statuteAccept" />
+          <FormCheckbox
+            label={personalConsentLabel}
+            name="personalDataAccept"
+          />
+          <FormCheckbox
+            label={additionalConsentLabel}
+            name="additionalAccept"
+          />
+        </div>
+        <div className="flex gap-4 items-center justify-center m-8">
+          <button type="submit" className="button-round button-filled ">
+            Submit
+          </button>
+        </div>
+      </form>
+      {/* <DevTool control={methods.control} /> */}
```

Comment:
What's this?

Reply:
Tool that helps to see real time changes in form. I'am using it for checking if everything is fine.